src/components/LMNotificationFeedItem/index.tsx

```diff
@@ -0,0 +1,119 @@
+import {View, TouchableOpacity, StyleSheet} from 'react-native';
+import React from 'react';
+import LMProfilePicture from '../../base/LMProfilePicture';
+import layout from '../../utils/layout';
+import LMText from '../../base/LMText';
+import {timeStamp} from '../../utils';
+import STYLES from '../../constants/constants';
+import {LMNotificationFeedItemProps} from './types';
+import LMIcon from '../../base/LMIcon';
+import {
+  DOCUMENT_ATTACHMENT_TYPE,
+  IMAGE_ATTACHMENT_TYPE,
+  VIDEO_ATTACHMENT_TYPE,
+} from '../../constants/strings';
+
+const LMNotificationFeedItem = ({
+  activity,
+  userProfilePicture,
+  boxStyle,
+  activityDateStyle,
+  activityTextStyle,
+  onMenuTap,
+  onTap,
+  menuIcon,
+}: LMNotificationFeedItemProps) => {
+  //creating profile picture props as per customization
+  const updatedProfilePictureProps = userProfilePicture
+    ? userProfilePicture
+    : {
+        fallbackText: activity.activityByUser.name,
+        size: 50,
+        imageUrl: activity.activityByUser.imageUrl,
+      };
+  return (
+    <View
+      style={StyleSheet.flatten([
+        styles.container,
+        boxStyle,
+        {backgroundColor: activity.isRead ? '' : STYLES.$COLORS.LIGHT_GREY},
+      ])}>
+      <TouchableOpacity onPress={onTap} style={{flexDirection: 'row'}}>
+        {/* profile avatar view */}
+        <View>
+          {/* profile picture section */}
+          <LMProfilePicture {...updatedProfilePictureProps} />
+          {/* handles the gallery and document icon on profile picture */}
+          {activity.activityEntityData?.attachments &&
```

Comment:
we can save `activity.activityEntityData?.attachments` in a variable and then use it.